docs/developer/database.md

```
@@ -16,3 +16,23 @@
 
 ### import_jobs
 - id, namespace, slug, status, progress
+
+### api_tokens
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)